bin/mail/2016-04-26-changeorg-petition/generate-lists.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US contacts
+us_contacts_query = %q(
+  !(role_s: student) && (
+    create_ip_country_s: "United States" ||
+    hoc_country_s: "us" ||
+    location_country_code_s: "US" ||
+    location_country_s: "United States"
```

Comment:
This is redundant, given that the following query returns zero results?
`"q": "!(role_s: student) && !(location_country_s: \"United States\") && (location_country_code_s: \"US\")"`

Aside, it may be useful to add a comment why all of these are necessary to grab the US contacts and why we aren't missing a fifth condition.